pyanaconda/payload/__init__.py

```diff
@@ -81,6 +81,10 @@ def __init__(self, data):
     def first_payload_reset(self):
         return self._first_payload_reset
 
+    @property
+    def is_hmc_enabled(self):
+        return self.data.method.method == "hmc"
```

Comment:
Just a note, the point of the `hmc` flag was to enable the selection of SE/HMC during the installation, so you can change the installation source to something different and then set SE/HMC again. I think that it will still work this way, because the spokes remember the initial value of this property, but it could be easy to miss it next time. It is not really documented or anything, so as long as the basic use case works, I am fine with it. It is just something to consider.

Reply:
You mean that the SE/HMC can change but the value should still be set to True. In other words, it can't be disabled?

Reply:
Thanks for the explanation.